### DISCUSSION TITLE
How to run angular serve with astrill.

### DISCUSSION THREAD

**Discussion by spangmae** [community/community#146109] [npm] [answered]
I am using astrill without browser only.
And I'd like to run angular project.
But I can't execute project and I am getting some errors.
How to fix it?

**Comment by Procaptain1029** [comment, marked as answer]
Hi, spangmae.
Please input this command.
ng serve --host 0.0.0.0
Then your problem will be solved.
Regard.

**Comment by IndianPhotoAcademy** [comment]
Configure Astrill to permit local network access to run ng serve in Angular while utilizing Astrill VPN. Check the "Allow LAN" or similar option in the Astrill settings. Run ng serve --host 0.0.0.0 to make the application accessible to other networked devices. A firewall rule should allow the connection.
